src/prob/pf.jl

```diff
@@ -730,7 +729,7 @@ function _cpf_load_interface(load::Dict{String,<:Any}, v_start::Dict{<:Any,<:Any
                 end
             elseif load["model"]==EXPONENTIAL
                 c_nl_func = function(v_bt)
-                    vd = v_bt[1:end-1].-v_bt[end]
+                    vd = v_bt[1:end-1].-v_bt[end]  # Is this correct?
```

Comment:
good point, we should know which terminal the neutral is, it's not neccessarily the last one. let's call to see if we can improve this. Same thing for generators.